app/src/main/java/com/randomappsinc/studentpicker/editing/EditNameListAdapter.java

```diff
@@ -75,46 +84,49 @@ public void importNamesFromList(ListInfo updatedListInfo) {
         setViews();
     }
 
+    @NonNull
     @Override
-    public int getCount() {
-        return content.getNumNames();
+    public NameViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext())
+                .inflate(R.layout.edit_person_name_cell, parent, false);
+        return new NameViewHolder(view);
     }
 
     @Override
-    public String getItem(int position) {
-        return content.getName(position);
+    public void onBindViewHolder(@NonNull NameViewHolder holder, int position) {
+        holder.loadName(position);
     }
 
     @Override
     public long getItemId(int position) {
         return getItem(position).hashCode();
     }
 
-    class NameViewHolder {
+    @Override
+    public int getItemCount() {
+        return content.getNumNames();
+    }
+
+    public String getItem(int position) {
+        return content.getName(position);
+    }
+
+    class NameViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener{
```

Comment:
Instead of doing this implements thing, let's stay consistent by continuing the leverage ButterKnife. Give the parent view of the cell an ID like "parent" and use @OnClick(R.id.parent).